skyrim-platform/src/platform_se/codegen/convert-files/Definitions.txt

```diff
@@ -319,6 +319,160 @@ export const enum MotionType {
     Character = 7
 }
 
+export const enum UiMenu {
+    Barter = 'BarterMenu',
+    Book = 'Book Menu',
+    Console = 'Console',
+    ConsoleNativeUI = 'Console Native UI Menu',
+    Container = 'ContainerMenu',
+    Crafting = 'Crafting Menu',
+    Credits = 'Credits Menu',
+    Cursor = 'Cursor Menu',
+    Debug = 'Debug Text Menu',
+    Dialogue = 'Dialogue Menu',
+    Fader = 'Fader Menu',
+    Favorites = 'FavoritesMenu',
+    Gift = 'GiftMenu',
+    HUD = 'HUD Menu',
+    Inventory = 'InventoryMenu',
+    Journal = 'Journal Menu',
+    Kinect = 'Kinect Menu',
+    LevelUp = 'LevelUp Menu',
+    Loading = 'Loading Menu',
+    Main = 'Main Menu',
+    Lockpicking = 'Lockpicking Menu',
+    Magic = 'MagicMenu',
+    Map = 'MapMenu',
+    MessageBox = 'MessageBoxMenu',
+    Mist = 'Mist Menu',
+    OverlayInteraction = 'Overlay Interaction Menu',
+    Overlay = 'Overlay Menu',
+    Quantity = 'Quantity Menu',
+    RaceSex = 'RaceSex Menu',
+    Sleep = 'Sleep/Wait Menu',
+    Stats = 'StatsMenu',
+    TitleSequence = 'TitleSequence Menu',
+    Top = 'Top Menu',
+    Training = 'Training Menu',
+    Tutorial = 'Tutorial Menu',
+    Tween = 'TweenMenu',
+}
+
+export const enum DxScanCodes {
```

Comment:
DxScanCodes -> DxScanCode (since we have Menu/MotionType and not Menus/MotionTypes)